run-tests.sh

```diff
@@ -39,6 +39,16 @@ npm install --unsafe-perm
 
 mkdir -p reports
 export JOBS=8
+export JUNIT_REPORT_PATH="reports/node$version/"
+export JUNIT_REPORT_STACK=1
+
+gsutil cp gs://grpc-testing-secrets/coveralls_credentials/grpc-node.rc .
```

Comment:
This can actually only work from Linux...

Reply:
How would you feel about separating the Linux and Mac test scripts?

Reply:
Not sure - it seems to be "pick your poison" to me. I don't like to burden the script with various platform-specific checks, but I also don't like the idea of duplicating code.

Note that we're already doing SOME platform-specific work in some of these scripts (https://github.com/grpc/grpc-node/blob/master/tools/release/kokoro.sh#L39) so maybe adding OS-specific code is a slightly less bitter pill to swallow ?

But I don't have any strong opinion one way or another.